Firestore/core/src/firebase/firestore/remote/serializer.h

```diff
@@ -346,6 +347,9 @@ class Serializer {
   std::unique_ptr<remote::WatchChange> DecodeDocumentChange(
       nanopb::Reader* reader,
       const google_firestore_v1_DocumentChange& change) const;
+  std::vector<model::TargetId> DecodeTargetIdArray(nanopb::Reader* reader,
```

Comment:
Nit: can you move this function down below this group and make it its own group (so that it is preceded and followed by an empty line)?